README.md

```diff
@@ -889,4 +890,3 @@ GNU General Public License for more details.
 
 You should have received a copy of the GNU General Public License
  along with PIMO.  If not, see <http://www.gnu.org/licenses/>.
-
```

Comment:
You should keep the last LF character, it is good practice (for diff readability when adding new line at the end of a file)